sdk/core/Azure.Core/src/Pipeline/HttpPipelineBuilder.cs

```diff
@@ -51,7 +52,8 @@ public static HttpPipeline Build(ClientOptions options, HttpPipelinePolicy[] per
 
             if (diagnostics.IsLoggingEnabled)
             {
-                policies.Add(new LoggingPolicy(diagnostics.IsLoggingContentEnabled, diagnostics.LoggingContentSizeLimit));
+                policies.Add(new LoggingPolicy(diagnostics.IsLoggingContentEnabled, diagnostics.LoggingContentSizeLimit,
+                    diagnostics.LoggingAllowedHeaders.ToArray(), diagnostics.LoggingAllowedQueryParameters.ToArray()));
```

Comment:
Can I pre-populate these in my ClientOptions .ctor if I wanted to provide a default allow-list?

Reply:
Yep, exactly.

Reply:
Should we have a default allow-list in Azure.Core?  Even if it's just things like `Date` and `x-ms-client-request-id`?